src/bundle/Resources/public/js/scripts/admin.location.adaptive.tabs.js

```diff
@@ -14,7 +14,7 @@
             </li>`
         );
     };
-    const primaryTabsList = doc.querySelector('.ez-tabs');
+    const primaryTabsList = doc.querySelector('.ez-tabs--adaptive');
```

Comment:
Why only tabs with modifier `--adaptive` can have "more"?
I think it worked on all tabs.